src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -885,7 +880,6 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 			return;
 		}
 		this.followupsDisposables.clear();
-		dom.clearNode(this.followupsContainer);
```

Comment:
We have empty followups stacking up now, why do you need this change?

Reply:
Not an intentional change, I've been having major issues with saving files, don't know how this happened